flake.nix

```diff
@@ -0,0 +1,134 @@
+{
+  description = "Website for ledger-cli.org";
+
+  nixConfig.bash-prompt = "ledger-cli.org$ ";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixpkgs-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem( system:
+    let
+      # NOTA BENE: Update the version and hash below for ledger and ledger-mode
+      # on new release to publish any changes to the documentation to ledger-cli.org.
+      dist = {
+        ledger = {
+          # TODO: Uncomment once ledger release contains changes from afh/ledger@web-docs
+          #version = "3.3.1";
+          #hash = "sha256-gBqf0ty8je912pZoupcA099515ngIK8BK0uk8nXz9M0=";
+          # TODO: Remove once ledger release contains changes from afh/ledger@web-docs
+          version = "7b7bb08f5c5903566f866af23acde6a3c451b7d3";
+          hash = "sha256-lZdHdjFO/RhtKX8/4QrnLpeF+cbXVkzz9JvNF6DAWE0=";
+        };
+        ledger-mode = {
+          version = "4.0.0";
+          hash = "sha256-YF7JfTe2GVWgb14XghZG0djbMGVzYtZmyjcBvHL02do=";
+        };
```

Comment:
I expect these should just be flake inputs. You can use tags there, for example.